core/arch/riscv64/mangle.c

```diff
@@ -32,6 +32,12 @@
 
 #include "../globals.h"
 #include "arch.h"
+#include "instr_create_shared.h"
```

Comment:
Do we need to comment the reason for including this header?

Reply:
Ah, that header is actually not needed for this PR. It's included for the `INSTR_*` and `XINST_*` macros, but this PR does not need these macros. I remove this line in https://github.com/DynamoRIO/dynamorio/pull/6388/commits/0403d68fc29718a11a2332d5c163b937f97246e4.